README.md

```diff
@@ -32,31 +31,64 @@ docker compose up build_deployment_package
 
 This will output the deployment zipfile in the root directory.
 
-## Running the Lambda locally ##
-
-The configuration in this repository allows you run the Lambda locally for
-testing as long as you do not need explicit permissions for other AWS
-services. This can be done with the following command:
-
-```console
-docker compose up run_lambda_locally
-```
-
-You can then invoke the Lambda using the following:
-
-```console
- curl -XPOST "http://localhost:9000/2015-03-31/functions/function/invocations" -d '{}'
+## Lambda inputs ##
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| account\_ids | The list of AWS accounts to query for egress IPs to publish. | `list(string)` | n/a | yes |
+| application\_tag | The name of the AWS tag whose value represents the application associated with an IP address. | `string` | `"Application"` | no |
+| bucket\_name | The name of the S3 bucket to publish egress IP address information to. | `string` | n/a | yes |
+| domain | The domain hosting the published file(s) containing egress IPs. | `string` | n/a | yes |
+| ec2\_read\_role\_name | The name of the IAM role that allows read access to the necessary EC2 attributes.  Note that this role must exist in each account that you want to query and it must be assumable by Lambda.  For an example policy, see [`cisagov/cool-accounts`](https://github.com/cisagov/cool-accounts/blob/develop/dynamic/ec2readonly_policy.tf). | `string` | `"EC2ReadOnly"` | no |
+| file\_configs | A list of objects that define the files to be published.  "app_regex" specifies a regular expression that is matched against the value of the application_tag to determine if the address should be included in the file.  "description" is the description of the published file.  "filename" is the name to assign the published file.  "static\_ips" is the list of CIDR blocks that will always be included in the published file. | `list(object({ app_regex = string, description = string, filename = string, static_ips = list(string) }))` | n/a | yes |
```

Comment:
I know this is copied from the Terraform project but I would prefer we be a little more Pythonic in our terminology (I'm not going to harass you to rewrite this using Python type hint syntax).

```suggestion
| file\_configs | A list of dictionaries that define the files to be published.  "app_regex" specifies a regular expression that is matched against the value of the application_tag to determine if the address should be included in the file.  "description" is the description of the published file.  "filename" is the name to assign the published file.  "static\_ips" is the list of CIDR blocks that will always be included in the published file. | `list(dict({ app_regex = string, description = string, filename = string, static_ips = list(string) }))` | n/a | yes |
```